modules/kafka/kafka.go

```diff
@@ -15,20 +15,37 @@ import (
 	"github.com/testcontainers/testcontainers-go/wait"
 )
 
-const publicPort = nat.Port("9093/tcp")
+const (
```

Comment:
suggest: these are expanding the API we have to maintain, lets keep them private.

Reply:
I made them public so the `kafka_test` package had access to them. It is a minor duplication without so I don't disagree with you here about moving them back to private and duplicating

Reply:
Yer that's the pain with the separate test package name, but better as you say to have the slight duplication to keep it clean.